src/EnergyPlus/SizingManager.cc

```diff
@@ -1334,10 +1334,10 @@ void ManageSystemSizingAdjustments(EnergyPlusData &state)
 
 void ManageSystemVentilationAdjustments(EnergyPlusData &state)
 {
-    auto &AirDistUnit(state.dataDefineEquipment->AirDistUnit);
-    auto &FinalSysSizing(state.dataSize->FinalSysSizing);
-    auto &VbzByZone(state.dataSize->VbzByZone);
-    auto &AirToZoneNodeInfo(state.dataAirLoop->AirToZoneNodeInfo);
+    auto &AirDistUnit = state.dataDefineEquipment->AirDistUnit;
```

Comment:
A reference is a shallow copy which by definition does not call a constructor. Don't use constructor notation.